### ISSUE TITLE
How can I preserve empty lines in HTML?

### ISSUE THREAD

**Issue by minthemiddle** [HookyQR/VSCodeBeautify#105]
* **Operating System (+Version)**: Sierra 
* **VS Code Version**:  1.9.1
* **beautify Version**:  0.8.1

Provide the settings you use: 
(VS Code workspace and user settings, .jsbeautifyrc, .editorconfig)
eg.:
VS Code:
```javascript
{
    "editor.formatOnSave": true,
    "files.insertFinalNewline": true,
    "html.format.preserveNewLines": true
}
```
.jsbeautifyrc
none

## Action performed

Separate HTML sections with emptylines, f.e.

```html
<header>

<nav>

    <ul>
        <li>…</li>
    </ul>

</nav>

</header>
```

## Expected results
Expected beautify to keep empty lines

## Actual results
All empty lines are removed

**Comment by HookyQR** [comment]
I don't think you can. Beautify tries to reduce file size while still keeping the structure tidy. Blank lines don't mean anything in HTML except in `pre`formatted block, so beautify removes the 'waste'.

**Comment by HookyQR** [comment]
There is an error in how I was copying `max_preserve_newlines` default from VS Code settings. Fixed in 0.8.3.

**Comment by minthemiddle** [comment]
Thanks, works as expected now. Beautification helps me to better read code, not to minify it. I use other tools to prepare my code for production. So now I can use your tool to help me write and read code better.
